practice-one/src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ContextMenu.css";
+import { Button } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { editIcon, deleteIcon } from "@assets/images";
+
+const ContextMenu = () => {
+  return (
+    <div className="context-menu">
+      <Button
+        variant={BUTTON_VARIANTS.EDIT}
+        icon={editIcon}
+        onClick={() => console.log("Clicked edit button")}
```

Comment:
create empty function or the props for it. Don't push debug log

Reply:
I let a empty function
![image](https://github.com/datdragon777/React-Training/assets/74806826/0ca7d112-bffd-497e-8e3a-de47bec32e13)